docs/src/lib/ClimaCoreTempestRemap.md

```diff
@@ -33,3 +33,82 @@ overlap_mesh
 remap_weights
 apply_remap
 ```
+
+# Example
+
+The following example converts an OrdinaryDiffEq solution object `sol` to a netcdf file, and remaps it to an regular latitude-longitude (RLL) grid.
+
+```julia
+using ClimaCore: Geometry, Meshes, Domains, Topologies, Spaces
+using NCDatasets, ClimaCoreTempestRemap
+
+# sol is the integrator solution
+# cspace is the center extrduded space
```

Comment:
Thanks @simonbyrne for adding this example! 
Is there a similar approach for our spherical 2D cases?

Reply:
yes, it should be basically the same, except that you don't need to worry about the staggering.

Reply:
Thanks. So in that case in the `def_space_coord` call below, one would just use the (unique) horizontal `space`, as 
```
def_space_coord(nc, space)
```